src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -45,4 +59,25 @@ public DataSet getDatasetWithPropertiesById(Integer datasetId) {
         return dataset;
     }
 
+    public List<DataSetProperty> processDatasetProperties(Integer datasetId, Date createDate, List<DataSetPropertyDTO> properties) {
```

Comment:
Minor suggestion, not required. Drop create date as an argument and instead create a new date inside this method.